plotille/_canvas.py

```diff
@@ -248,8 +268,20 @@ def plot(self, linesep=linesep):
             linesep: str  The requested line seperator. default: os.linesep
 
         Returns:
-            unicode: The canvas as a string.
+            unicode: The canvas as a string, overlaid with textual elements
         """
 
-        return linesep.join(''.join(map(six.text_type, row))
-                            for row in reversed(self._canvas))
+        # classic:
+        # ret = linesep.join(''.join(map(six.text_type, row)) for row in reversed(self._canvas))
+        # return ret
+
+        # combine _canvas (Dots) and overlay (text) so that overlay is on top of dots
+        ret_combined = ''
+        for row_index in range(len(self._canvas))[::-1]:
+            for ii in range(len(self._canvas[row_index])):
+                if(self._canvas_overlay[row_index][ii] == ' '):
```

Comment:
This means, if the text in the overlay has a `' '`, we might have dots between the text?